File: w01/d04/Wake_Lankard/calculator/calculator.rb
class Calculator
  def add(x,y)
    x+y
  end

  def subtract(x,y)
    x-y
  end

  def divide(x,y)
    return "NaN" if y == 0
    x.to_f/y.to_f
  end

  def multiply(x,y)
    x*y
  end



end

File: w06/d01/Jeff_Winkler/calculator.js

function calculator(x, y, operation) {
  switch (operation){
  case 'add':
    console.log(x+y);
    break;
  case 'subtract':
    console.log(x-y);
    break;
  case 'multiply':
    console.log(x*y);
    break;
  case 'divide':
    console.log(x/y);
    break;
  default:
    console.log('Please provide add, subtract, multiply, or divide.');
  }
}

calculator(2,3,'add');
calculator(2,3,'subtract');
calculator(2,3,'multiply');
calculator(2,3,'divide');


